heron/uploaders/src/java/com/twitter/heron/uploader/docker/sample.yaml

```diff
@@ -0,0 +1,11 @@
+# uploader class for transferring the topology jar/tar files to storage
+heron.class.uploader:         com.twitter.heron.uploader.docker.DockerUploader
+
+#base image
+heron.uploader.docker.base:                      ubuntu:latest
```

Comment:
If you need the heron binaries in a container, we can build it as a part of the release process and ship it. Let me know if this will simplify it.